### PR TITLE
Feat/basic navigation

### PR DESCRIPTION
**Pull request by arthurnog**

### CODE REVIEW

File: packages/app/features/settings/screen.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { View, StyleSheet, Text } from 'react-native'
+import React from 'react'
+
+export { SettingsScreen as default } from 'app/features/settings/screen'
+
+
+export function SettingsScreen() {
```
Comment (FelipeEmos) [inline packages/app/features/settings/screen.tsx]: I know that adding a `SettingsScreen` was the tutorial, but we don't need that.
What about removing it? 🙂

File: package.json [hunk 2]
```diff
@@ -10,11 +10,15 @@
...
   "scripts": {
     "native": "cd apps/expo && expo start",
     "web": "cd apps/next && yarn next"
   },
-  "packageManager": "yarn@3.4.1"
+  "packageManager": "yarn@3.4.1",
+  "dependencies": {
+    "react-native": "^0.71.4",
+    "ts-node": "^10.9.1"
+  }
```
Comment (FelipeEmos) [inline package.json:23]: If you change a dependency or a dependency version, put that in a separate `chore` commit

**Review by FelipeEmos — COMMENTED** [review]
About each commit...
- First commit: It's a `wip`, you shouldn't have wip commits in a PR that's opened. That's because it may enter the code base and we don't want a `wip` commit in the codebase.

- Second commit: It's a `fix` for something you didn't do correctly in the first commit ( let's put it that way ). Then the best approach is to correct the first commit... because the PR is still opened and we can deliver a more clean sequence of commits in that way. To modify commits from the past you should use `git rebase -i`, it's something worth doing a videocall with you unless you are very familiar with it.

- Last commit: It's not a `chore`, you are not changing dependencies. That's a feature, because you are adding new funcitonality (new screens) 🙂

**Comment by FelipeEmos** [comment]
*nit*: You also could change the title of the PR to something a little different
`feat(navigation): Add all screens` , which is closer to the conventional commit pattern and is more descriptive

File: packages/app/features/settings/screen.tsx (hunk 1 above)
Comment (arthurnog) [reply packages/app/features/settings/screen.tsx]: Removed

File: package.json (hunk 2 above)
Comment (arthurnog) [reply package.json:23]: Reseted the branch and created a chore commit for this

File: packages/app/features/home/screen.tsx [hunk 3]
```diff
@@ -9,42 +10,33 @@ export function HomeScreen() {
...
       sx={{ flex: 1, justifyContent: 'center', alignItems: 'center', p: 16 }}
     >
-      <H1 sx={{ fontWeight: '800' }}>Welcome to Solito.</H1>
+      <H1 sx={{ fontWeight: '800' }}>Tic Tac Tuts</H1>
       <View sx={{ maxWidth: 600 }}>
         <P sx={{ textAlign: 'center' }}>
           Here is a basic starter to show you how you can navigate from one
           screen to another. This screen uses the same code on Next.js and React
           Native.
         </P>
```
Comment (FelipeEmos) [inline packages/app/features/home/screen.tsx]: At some point you'll have to remove this, so why don't do it now? 😅 

It's up to you

**Review by FelipeEmos — APPROVED** [review]
Nice!

File: packages/app/features/home/screen.tsx (hunk 3 above)
Comment (arthurnog) [reply packages/app/features/home/screen.tsx]: Did an amend

**Pull request merged**